dev/environment-dev.yml

```diff
@@ -0,0 +1,22 @@
+# You can use this file to create a conda environment for this pipeline:
```

Comment:
Hm that looks good to me - so its probably a problem to get the OpenMS recipe on Bioconda to work with all the dependencies again (remember we had issues at some point with this ...)

Reply:
I could hide the profile, so that only "developers" can enable it? In any case, one has to build the container by hand anyway. If they do that, they probably know what to look for, to check if it is a malicious container.

Reply:
I guess the move to handle this differently in the future is a good one - might be fine for this first release & then move towards versioned releases on bioconda at some point :-)